packages/account/src/Components/additional-kyc-info-modal/additional-kyc-info-form.tsx

```diff
@@ -0,0 +1,248 @@
+import React from 'react';
+import {
+    Autocomplete,
+    Button,
+    DesktopWrapper,
+    Loading,
+    MobileWrapper,
+    Modal,
+    SelectNative,
+    Text,
+} from '@deriv/components';
+import { observer, useStore } from '@deriv/stores';
+import { Localize } from '@deriv/translations';
+import classNames from 'classnames';
+import { isMobile } from '@deriv/shared';
+import { Field, FieldProps, Form, Formik, FormikErrors } from 'formik';
+import FormFieldInfo from '../form-field-info';
+import { FormInputField } from '../forms/form-fields';
+import { TListItem, getFormConfig } from './form-config';
+import { useSettings } from '../../../../api/src/hooks';
+
+const FormTitle = () => (
+    <Text
+        as='p'
+        size='s'
+        line_height='xxl'
+        align={isMobile() ? 'left' : 'center'}
+        className='additional-kyc-info-modal__form--header'
+    >
+        <Localize i18n_default_text='Please take a moment to update your information now.' />
+    </Text>
+);
+
+type TAdditionalKycInfoFormProps = {
+    setError?: React.Dispatch<React.SetStateAction<string>>;
+};
+
+export const AdditionalKycInfoForm = observer(({ setError }: TAdditionalKycInfoFormProps) => {
+    const { client, ui, notifications } = useStore();
+    const { residence_list, updateAccountStatus } = client;
+    const {
+        update,
+        mutation: { isLoading, isSuccess, error, isError },
+        data,
+        isLoading: isAccountSettingsLoading,
+    } = useSettings();
+
+    const { fields, initialValues, validate } = getFormConfig({
+        account_settings: data,
+        residence_list,
+        required_fields: ['place_of_birth', 'tax_residence', 'tax_identification_number', 'account_opening_reason'],
+    });
+
+    const onSubmit = (values: typeof initialValues) => {
+        const tax_residence = residence_list.find(item => item.text === values.tax_residence)?.value;
+        const place_of_birth = residence_list.find(item => item.text === values.place_of_birth)?.value;
+
+        update({ ...values, tax_residence, place_of_birth });
+    };
+
+    React.useEffect(() => {
+        if (isSuccess) {
+            updateAccountStatus();
+            notifications.refreshNotifications();
+            ui.toggleAdditionalKycInfoModal();
+            ui.toggleKycInformationSubmittedModal();
+        }
+    }, [isSuccess, notifications, ui, updateAccountStatus]);
```

Comment:
did you intent to listen to notifications here?
asking because `notifications` store has `notifications` array inside it.